packages/mui-base/src/useSlider/useSlider.ts

```diff
@@ -259,7 +253,8 @@ export function useSlider(parameters: UseSliderParameters): UseSliderReturnValue
 
   const range = Array.isArray(valueDerived);
   let values = range ? valueDerived.slice().sort(asc) : [valueDerived];
-  values = values.map((value) => clamp(value, min, max));
+  values = values.map((value) => (typeof value === 'number' ? clamp(value, min, max) : min));
```

Comment:
No need to verify that the value type is `number`; TypeScript already ensures it's a `number` type, and PropTypes warn in JavaScript. I also noticed that the condition in `clamp`:
```js
if (value == null) {
    return min;
}
```
was actually redundant, as it's already covered by falling back to `min` [here](https://github.com/mui/material-ui/blob/master/packages/mui-base/src/useSlider/useSlider.ts#L237).